<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-1DSDOME4" failures="1" tests="12" name="Functionone" time="0.07" errors="0" timestamp="2021-08-22T20:04:46 IST">
  <testcase classname="CrossCutting.Finance" name="CCSucess" time="0.005"/>
  <testcase classname="CrossCutting.Finance" name="CCrejection" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at CrossCutting.Finance.CCrejection(Finance.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- CCrejection -->
  <testcase classname="CrossCutting.Finance" name="FinaceCC" time="0.001"/>
  <testcase classname="CrossCutting.Finance" name="FinanceDD" time="0.001"/>
  <testcase classname="CrossCutting.Finance" name="FinancePS" time="0.001"/>
  <testcase classname="CrossCutting.DocumentManagement" name="DocBatch" time="0.002"/>
  <testcase classname="CrossCutting.DocumentManagement" name="DocCreatecm" time="0.001"/>
  <testcase classname="CrossCutting.DocumentManagement" name="DocIPP" time="0.001"/>
  <testcase classname="CrossCutting.DocumentManagement" name="Paymentsch" time="0.001"/>
  <testcase classname="CrossCutting.DocumentManagement" name="Paymentsch" time="0.001"/>
  <testcase classname="CrossCutting.DocumentManagement" name="Paymentsch" time="0.001"/>
  <testcase classname="CrossCutting.DocumentManagement" name="Sepa" time="0.001"/>
</testsuite> <!-- Functionone -->
